go.mod

```diff
@@ -59,6 +59,15 @@ require (
 	github.com/kolide/systray v1.10.4
 	github.com/kolide/toast v1.0.0
 	github.com/shirou/gopsutil/v3 v3.23.3
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.16.0
+)
+
+require (
+	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
+	github.com/felixge/httpsnoop v1.0.3 // indirect
```

Comment:
😦  🤷

Reply:
Like I said 🤷 

Gotta just live with the extra dependancies. I guess `backoff` is coming from some notary thing